src/devicecontroller/NoOpDeviceController.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import { EventController } from '..';
```

Comment:
Should this import come from an absolute path like others instead of relative import from `src/index.ts`?

Reply:
Yes this should be, nice catch!